scripts/install-nix-from-closure.sh

```diff
@@ -52,8 +43,8 @@ elif [ "$(uname -s)" = "Linux" ]; then
     echo "Note: a multi-user installation is possible. See https://nixos.org/nix/manual/#sect-multi-user-installation" >&2
 fi
 
-INSTALL_MODE=no-daemon
-CREATE_DARWIN_VOLUME=0
+export INSTALL_MODE=no-daemon
+CREATE_DARWIN_VOLUME=${CREATE_DARWIN_VOLUME:-1} # now default
```

Comment:
We can reduce name duplication using `:=` instead

```suggestion
: ${CREATE_DARWIN_VOLUME:=1} # now default
```

Reply:
clarity/consistency w/ other installer code on this one